server/server.js

```diff
@@ -27,10 +34,114 @@ app.use(express.json());
 //     res.sendFile(path.join(__dirname, '../client/dist', 'index.html'))
 // })
 
-//All Planned Routes
+//----------------------------------- All Routes --------------------------------
+
+//********* ACCOUNT AND LOG IN SECTION **********
+
+//Function to check access tokens before authorizing the route
```

Comment:
Something to implement in the future, creating a routes folder to hold files for specific routes such as having one for all auth (login/register) related routes, another for all pet info related routes, etc. This could help maintain organization among your endpoints when you need to go back and change something in the future or if you end up having others collaborate with you as well!

Reply:
SO helpful! It's definitely getting crowded in there. I've added this to my post-grad to-do list. Thanks for the help and wonderful feedback, Abbie!